applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -14,7 +15,7 @@
 #include <../subsys/bluetooth/audio/bap_endpoint.h>
 
 #include "macros_common.h"
-#include "ctrl_events.h"
+#include "nrf5340_audio_common.h"
```

Comment:
Should nrf5340_audio_common.h be on top? Same for several of the other files

Reply:
I don't think so. My view is that it should be:

* specific header for that file (le_auido.h in this case)
* c standard stuff if needed (stdlib.h etc)
* zephyr specific header files
* application related header files

The order of the header files in each block shouldn't matter, but personally I like them to be ordered by number of characters, but I won't go out of my way to reorder just to suit my OCD :P